Translations/Localization.md

```diff
@@ -0,0 +1,108 @@
+> Localization describes the subsequent process of translating an internationalized theme/plugin. Localization is often abbreviated as l10n (because there are 10 letters between the l and the n.)
+
+This is a handy up-to-date how-to for generating WordPress Theme/Plugin translation files with WP-CLI. The following workflow is covered: creating a POT (Portable Object Template) file, copying it to a PO (Portable Object) file where the `msgstr` sections are translated or updating pre-existing PO file and finally building the MO (Machine Object) file which is used by `gettext` functions to output translations to the user.
```

Comment:
```suggestion
This is a handy up-to-date how-to for generating WordPress theme/plugin translation files with the WP CLI.

 The following workflow is covered: creating a POT (Portable Object Template) file, copying it to a PO (Portable Object) file where the `msgstr` sections are translated or updating pre-existing PO file and finally building the MO (Machine Object) file which is used by `gettext` functions to output translations to the user.
```

Reply:
Actually, I need to add the creation of JED files here as well :)